usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -254,26 +238,40 @@ def handle_column(col, bronze_table_name):
-                    CASE WHEN regexp({bronze_table_name}.{col.source}, '{regexp_mmddYYYY}')
-                              THEN CAST((regexp_extract({bronze_table_name}.{col.source}, '{regexp_mmddYYYY}', 5)
-                                         || '-' ||
-                                         regexp_extract({bronze_table_name}.{col.source}, '{regexp_mmddYYYY}', 1)
-                                         || '-' ||
-                                         regexp_extract({bronze_table_name}.{col.source}, '{regexp_mmddYYYY}', 3))
-                                        AS DATE)
-                         ELSE CAST((regexp_extract({bronze_table_name}.{col.source}, '{regexp_YYYYmmdd}', 1)
-                                    || '-' ||
-                                    regexp_extract({bronze_table_name}.{col.source}, '{regexp_YYYYmmdd}', 3)
-                                    || '-' ||
-                                    regexp_extract({bronze_table_name}.{col.source}, '{regexp_YYYYmmdd}', 5))
-                                   AS DATE)
-                    END AS {col.dest_name}
-                """
-            elif col.handling == "truncate_string_date":
-                # These are string fields that actually hold DATES/TIMESTAMPS, but need the non-DATE part discarded,
-                # even though they remain as strings
-                return f"truncate_string_date({bronze_table_name}.{col.source}) AS {col.dest_name}"
+                if col.handling == "parse_string_date":
+                    return f"""
+                        CASE WHEN regexp({bronze_table_name}.{col.source}, '{regexp_mmddYYYY}')
+                                  THEN CAST((regexp_extract({bronze_table_name}.{col.source}, '{regexp_mmddYYYY}', 5)
+                                             || '-' ||
+                                             regexp_extract({bronze_table_name}.{col.source}, '{regexp_mmddYYYY}', 1)
+                                             || '-' ||
+                                             regexp_extract({bronze_table_name}.{col.source}, '{regexp_mmddYYYY}', 3))
+                                            AS DATE)
+                             ELSE CAST((regexp_extract({bronze_table_name}.{col.source}, '{regexp_YYYYmmdd}', 1)
+                                        || '-' ||
+                                        regexp_extract({bronze_table_name}.{col.source}, '{regexp_YYYYmmdd}', 3)
+                                        || '-' ||
+                                        regexp_extract({bronze_table_name}.{col.source}, '{regexp_YYYYmmdd}', 5))
+                                       AS DATE)
+                        END AS {col.dest_name}
+                    """
+                else:
+                    # These are string fields that actually hold DATES/TIMESTAMPS, but need the non-DATE part discarded,
+                    # even though they remain as strings
+                    return f"""
```

Comment:
What do you think about `parse_string_date_truncate_time` as an alternative to `truncate_string_date`?